### DISCUSSION TITLE
歡迎來到會考沙漏的官方討論區

### DISCUSSION THREAD

**Discussion by SiongSng** [SiongSng/cap-countdown#8] [公告]
## 👋 歡迎！
我們希望這裡能成為您與社群其他成員交流的地方。我們希望您能：

  * 提出您想知道的問題
  * 分享您的想法
  * 與其他社群成員互動
  * 歡迎他人並保持開放的心態。請記住，這是一個我們一起建立的社群💪。

心動不如馬上行動，歡迎在下面留言介紹自己，並告訴我們您想在這個社群中做什麼事情。

**Comment by app97204** [comment]
這個app很棒

**Reply by SiongSng** [reply]
感謝您對會考沙漏的支持！

**Reply by app97204** [reply]
以後會提供其他年份的試題嗎

**Reply by SiongSng** [reply]
當然會喔！目前規劃優先新增 110 年與十幾天後即將到來的 112 會考。

**Reply by app97204** [reply]
是否考慮使用markdown 格式處理特殊文章呢？
目前圖片格式觀看不易
https://pub.dev/packages/flutter_markdown

**Reply by app97204** [reply]
還有一個問題是作答完畢後不小心退出資料無法還原

**Reply by SiongSng** [reply]
> 是否考慮使用markdown 格式處理特殊文章呢？
> 目前圖片格式觀看不易
> https://pub.dev/packages/flutter_markdown

可以請您截圖指出具體造成觀看不易的題目嗎？
還是您是指透過 Markdown 將圖片插在題目中？

**Reply by SiongSng** [reply]
> 還有一個問題是作答完畢後不小心退出資料無法還原

這部分我目前還沒想到更好的解方

**Reply by app97204** [reply]
![image](https://user-images.githubusercontent.com/92156698/236660067-9f7ab835-eaff-4b8f-95d6-2fa885d7a645.png)![image](https://user-images.githubusercontent.com/92156698/236660070-13b5a413-829d-4a3f-a5b1-1a64b8f62835.png)
從右到左時很容易對錯行，md的意思是把文字已md排版直接顯示，不使用圖片，好處是可以符合螢幕寬度

**Reply by SiongSng** [reply]
喔我懂你的意思了，這部分技術上我需要做一些研究來搭配 MD ，因為目前為了支援特殊數學式與化學式已經用了一種格式化文字的技術（LaTeX）

**Reply by SiongSng** [reply]
感謝您的建議，新版本現已支援更豐富的題目佈局樣式，使作答體驗更為逼真。
